clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiConverter.groovy

```diff
@@ -895,38 +930,74 @@ class KubernetesApiConverter {
       .endMetadata()
       .withNewSpec()
 
-    if (description.restartPolicy) {
-      podTemplateSpecBuilder.withRestartPolicy(description.restartPolicy)
-    }
+    KubernetesPodSpecDescription podSpec = new KubernetesPodSpecDescription()
+    podSpec = description.podSpec
 
-    if (description.terminationGracePeriodSeconds) {
-      podTemplateSpecBuilder.withTerminationGracePeriodSeconds(description.terminationGracePeriodSeconds)
-    }
+    if ( podSpec ) {
```

Comment:
There is another UI developer is working on UI changes.  The code should be running fine for now with original UI code base.